### PR TITLE
LKE-3681 warn user about size

### PR DESCRIPTION
**Pull request by aminerhanemi**

### CODE REVIEW

**Review by maxime-allex — APPROVED** [review]

File: src/public/index.ts [hunk 1]
```diff
@@ -54,8 +59,12 @@ function main() {
   const readButton = document.getElementById("readButton") as HTMLElement;
   const showExampleButton = document.getElementById("showExampleButton") as HTMLElement;
   fileInput.addEventListener("change", uploader.showFile.bind(uploader));
-  readButton.addEventListener("click", () => {
-    uploader.readFile();
+  readButton.addEventListener("click", async () => {
+    await uploader.readFile();
```
Comment (EricMallard) [inline src/public/index.ts:63]: would it make sense to make readFile return an object with all those properties below (sourceKey, propertiesValue, etc)? I see it as more 'natural' than creating getters to those properties (not blocking just a suggestion)

Comment (aminerhanemi) [reply src/public/index.ts:63]: good point, I was taking to do it. not sure why I changed my mind or  I think just forgot to do the change.

File: src/public/index.ts [hunk 2]
```diff
@@ -60,11 +60,11 @@ function main() {
   const showExampleButton = document.getElementById("showExampleButton") as HTMLElement;
   fileInput.addEventListener("change", uploader.showFile.bind(uploader));
   readButton.addEventListener("click", async () => {
-    await uploader.readFile();
-    sourceKey = uploader.sourceKey;
-    propertiesValue = uploader.propertiesValue;
-    propertiesName = uploader.propertiesName;
-    entityName = uploader.entityName;
+    const fileData = await uploader.readFile();
```
Comment (EricMallard) [inline src/public/index.ts:63]: can we use destructuring here?

Comment (aminerhanemi) [reply src/public/index.ts]: done

**Review by EricMallard — APPROVED** [review]
Great!

**Pull request merged**